src/contracts/core/DelegationManager.sol

```diff
@@ -535,137 +572,151 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
             );
         }
 
+        uint32 epochForEndOfSlashability = EpochUtils.getEndOfSlashabilityEpoch(withdrawalCreationEpoch(withdrawalRoot));
+        require(EpochUtils.currentEpoch() > epochForEndOfSlashability,
+            "DelegationManager._completeQueuedWithdrawal: withdrawal is still slashable");
```

Comment:
I feel like this shouldn't apply for stakers that weren't delegated to an operator when they started withdrawing.